src/tests/unit/test_lib.py

```diff
@@ -16,6 +16,32 @@ def test_install_systemd(self, weechat):
             content = service_file.read()
             assert 'ExecStart=/usr/bin/screen -D -m -S weechat /usr/bin/weechat' in content
 
+    def test_install_weeslack(self, weechat, mock_install_remote,
+                              mock_symlink, mock_chownr, mock_isfile,
```

Comment:
Same comment as above if you can check the Mock objects with out applying the fixtures again.